tests/helper/tests/ssh_authorized.py

```diff
@@ -1,6 +1,6 @@
 import hashlib
 
-def ssh_authorized(client, testconfig):
+def ssh_authorized(client, testconfig, chroot):
```

Comment:
This test has originally been used only as an unit-test in a chroot context, so I would stick to this because which users exists during image runtime heavily depends on the specific cloud provider and the initialization they are using there (e.g. cloud-init). 

A whitelist could help but it must be checked if the amount of users is static enough to create proper whitelists for each cloud provider and I would suggest to do it in a dedicated issue because as of now, we do not have a requirement to let this test be executed during runtime but it should be executed at least during build time which is still the case.